hooks/base16-vim.fish

```diff
@@ -20,12 +20,12 @@ end
 # ----------------------------------------------------------------------
 read current_theme_name < "$BASE16_SHELL_THEME_NAME_PATH"
 
-set vim_output $(string join \
+set vim_output (string join \
```

Comment:
I've just noticed another bug here. Can you remove the new line escaped characters on the next 2 lines since this change adds them?

```suggestion
set vim_output (string join "\n" \
```

Reply:
No problem!